ydb/docs/en/core/yql/reference/yql-core/syntax/create_view.md

```diff
@@ -0,0 +1,88 @@
+# CREATE VIEW
+
+CREATE VIEW defines a view of a query.
+
+A view is a logical representation of a table formed by the specified query. The view does not physically store the table, but runs the query to produce the data whenever the view is selected from.
+
+## Syntax
+
+```sql
+CREATE VIEW name
+[ WITH ( view_option_name [= view_option_value] [, ... ] ) ]
+AS query
+```
+
+### Parameters
+
+* `name`
+
+The name of the view to be created. The name must be distinct from the name of any other schema object.
+* `query`
+
+The SELECT query, which will be used to produce the logical table the view represents.
+
+`WITH ( view_option_name [= view_option_value] [, ... ] )`
+
+This clause specifies optional parameters for a view. The following parameters are supported:
+
+* `security_invoker` (Bool) {#security_invoker}
+
+This option causes the underlying base relations to be checked against the privileges of the user of the view rather than the view owner.
+
+## Notes {#notes}
+
+`security_invoker` option must be always set to true, because the default behavior for views is to execute the query on behalf of the view's creator, which is not supported yet.
+
+The execution context of the view's query is different from the context of the enclosing SELECT from the view statement. It does not "see" previously defined PRAGMAs like TablePathPrefix, named expressions, etc. Most importantly, users must specify the tables (or views) they select from in the view's query by their schema-qualified names. You can see in the [examples](#examples) that the absolute path like `/domain/database/path/to/underlying_table` is used to specify the table, from which a view selects. The particular context of the view's query compilation might change in next releases.
+
+If you wish to specify column names that you would like to see in the output of the view, you might do so by modifying the view's query:
+```sql
+CREATE VIEW view_with_a_renamed_column WITH (security_invoker = TRUE) AS
+    SELECT
+        original_column_name AS custom_column_name
+    FROM `/domain/database/path/to/underlying_table`;
+```
+
+Star (`*`) expansion in the view's query happens each time you read from the view. The list of columns returned by the following statement:
+```sql
+/*
+CREATE VIEW view_with_a_star WITH (security_invoker = TRUE) AS
+    SELECT
+        *
+    FROM `/domain/database/path/to/underlying_table`;
```

Comment:
```suggestion
SELECT *
FROM `/domain/database/path/to/underlying_table`;
```

Reply:
I agree with the suggested change in indentation, but I believe the asterisk should be located on a separate line according to the SQL [format tool](https://github.com/ydb-platform/ydb/blame/fd412ef41d0fef714ca8646c0b78f023acc593e0/ydb/library/yql/sql/v1/format/sql_format_ut.cpp#L760) of YDB